Flick/Controllers/ListViewController.swift

```diff
@@ -300,7 +300,10 @@ extension ListViewController: UICollectionViewDelegateFlowLayout {
         case .listSummary:
             return CGSize(width: collectionView.frame.width, height: listSummaryHeight)
         case .mediaList:
-            let width = (mediaCollectionView.frame.width - 2 * (cellPadding + edgeInsets)) / 3.0
+            let numCellsInRow: CGFloat = 3
+            let flowLayout = collectionViewLayout as! UICollectionViewFlowLayout
+            let extraSpace = 2 * edgeInsets + (flowLayout.minimumInteritemSpacing * numCellsInRow)
```

Comment:
do we not have access to the `minimumInteritemSpacing` since its a value that we set when we created the flow layout? just wondering if it is necessary to get the layout object, cast it as a `collectionviewflowlayout`, and then get `minimumInteritemSpacing`